Tests/MapLibreSwiftUITests/Support/XCTestAssertView.swift

```diff
@@ -1,25 +1,23 @@
+import MapLibreSwiftUI
+import SnapshotTesting
 import SwiftUI
 import XCTest
-import SnapshotTesting
-import MapLibreSwiftUI
 
 // TODO: This is a WIP that needs some additional eyes
 extension XCTestCase {
-    
-    func assertView<Content: View>(
+    func assertView(
         named name: String? = nil,
         record: Bool = false,
         frame: CGSize = CGSize(width: 430, height: 932),
-        expectation: XCTestExpectation? = nil,
-        @ViewBuilder content: () -> Content,
+        expectation _: XCTestExpectation? = nil,
```

Comment:
I guess we don't actually use this parameter?

Reply:
I'm just gonna leave this for now. Once we figure out how to asynchronously snapshot test the MapView after it's loaded this will be corrected.